tests/assets/video/Big_Buck_Bunny_1080_1s_av1.mp4

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
We should rename `.github/workflows/png_only_on_lfs.yml` and make sure it also checks for mp4